lib/transition/import/hits.rb

```diff
@@ -64,9 +64,9 @@ def self.from_tsv!(filename)
 
           Hit.transaction do
             import_record = ImportedHitsFile.where(
-                filename: relative_filename).first_or_initialize
+              filename: relative_filename).first_or_initialize
 
-            job.skip! and next if import_record.same_on_disk?
+            job.skip! && next if import_record.same_on_disk?
```

Comment:
Seems we can mutate the cop to only look at conditionals (see https://github.com/bbatsov/rubocop/blob/bcb42a2e5960c56ef0a1c8b4b4093ae5b3e71ce8/config/default.yml#L185-L191).  I'll make that change locally, revert, and re-run see what happens.